pandas/tests/extension/test_datetime.py

```diff
@@ -98,6 +98,20 @@ def test_combine_add(self, data_repeated):
         # Timestamp.__add__(Timestamp) not defined
         pass
 
+    @pytest.mark.parametrize(
+        "method", ["argmax", "max", "argmin", "min"],
+    )
```

Comment:
why is this not in the base tests?

Reply:
The `min` and `max` methods of `DatetimeArray` are already defined prior to this PR. The behavior of empty `DatetimeArray` is to return `NaT` other than raising `ValueError`. (Raising `ValueError` is the behavior expected in base test.)

Reply:
What is the minimally invasive change that would make the DTA (and probably TDA and PA) method consistent with everything else?

Reply:
What is the minimally invasive change that would make the DTA (and probably TDA and PA) method consistent with everything else?

Reply:
Do we want to deprecate this behavior in favor of raising then?